buf/registry/module/v1beta1/label_service.proto

```diff
@@ -157,12 +157,56 @@ message ListLabelHistoryRequest {
 }
 
 message ListLabelHistoryResponse {
+  // Whether the Commit was a breaking change.
+  enum BreakingState {
+    BREAKING_STATE_UNSPECIFIED = 0;
+    // The Commit did not represent a breaking change.
+    BREAKING_STATE_NONE = 1;
+    // The Commit represented a breaking change at the WIRE level.
+    BREAKING_STATE_WIRE = 2;
+    // The Commit represented a breaking change at the WIRE_JSON level.
+    BREAKING_STATE_WIRE_JSON = 3;
+    // The Commit represented a breaking change at the PACKAGE level.
+    BREAKING_STATE_PACKAGE = 4;
+    // The Commit represented a breaking change at the FILE level.
+    BREAKING_STATE_FILE = 5;
+  }
+  // The Commit's state in the review flow.
+  enum ReviewState {
+    REVIEW_STATE_UNSPECIFIED = 0;
+    // Review was not enabled at the time this Commit was uploaded.
+    REVIEW_STATE_DISABLED = 1;
+    // The Commit was reviewed and was accepted.
+    REVIEW_STATE_ACCEPTED = 2;
+    // The Commit was reviewed and was rejected.
+    REVIEW_STATE_REJECTED = 3;
+    // The Commit has not yet been reviewed and is pending.
+    REVIEW_STATE_PENDING = 4;
+  }
+  message Value {
+    // The Commit.
+    Commit commit = 1 [(buf.validate.field).required = true];
+    // The Commit's breaking state.
+    //
+    // This field will only be returned if breaking change detection is enabled.
+    //
+    // Breaking change detection is an enteprise-only feature - contact us to
+    // learn more!
+    BreakingState breaking_state = 2 [(buf.validate.field).enum.defined_only = true];
```

Comment:
Should this be repeated? If we keep it singular, then client code needs to encode the hierarchy of breaking categories (e.g., that breaking at the `PACKAGE` level implies breaking at the `FILE` level). If we make it repeated, the server can respond with _all_ the categories (e.g., `[PACKAGE, FILE]`) and the clients can stay dumb.

Reply:
Hmm I don't know, I don't feel too strongly either way. Arguments against `repeated`:

- Hierarchy is pretty-well understood and agreed upon since the beginning of  `buf`.
- Making this `repeated` enables states that are incorrect (even if we never respond to them) - what does it mean to have `[NONE, WIRE]`? What does it mean to have `[WIRE, FILE]` but not `WIRE_JSON` or `PACKAGE`?

I might lean slightly against `repeated`, but don't know.